dbt/include/bigquery/macros/adapters.sql

```diff
@@ -99,6 +99,7 @@
 {% macro bigquery__persist_docs(relation, model, for_relation, for_columns) -%}
   {% if for_columns and config.persist_column_docs() and model.columns %}
     {% do alter_column_comment(relation, model.columns) %}
+    {% do adapter.update_table_description(relation, model.description) %}
```

Comment:
currently this is within the `if` that has to do with `column` comments.

perhaps there needs to be a new `if` that is something like the below. that said, I'm very confused about the purpose of `for_relation` and `for_columns`
```suggestion
  {% if for_relation and config.persist_relation_docs() and model.description %}
    {% do adapter.update_table_description(relation, model.description) %}
  {% endif %}
```

Actually it looks like the `seed` materialization is already doing something very similar. https://github.com/dbt-labs/dbt-bigquery/blob/978a1aea7d18af79d6d78d0ab5e8d058c4b7c93c/dbt/include/bigquery/macros/materializations/seed.sql#L20-L23

Reply:
@dataders 
Thank you for your comment.

I modified the implementation to use an existing method in the seed materialization.
- https://github.com/dbt-labs/dbt-bigquery/pull/1139/commits/ac75ff21a4432627aa218a810af8e7383101daa3

If I change the method `update_table_description` itself, I will have to modify the two place, 
1. dbt/include/bigquery/macros/adapters.sql
2. dbt-bigquery/dbt/include/bigquery/macros/materializations/seed.sql

Should I do this in this PR?

Reply:
@dataders let me remind you

Reply:
@dataders let me remind you

Reply:
@colin-rogers-dbt 

could you help me?